subreddit.py

```diff
@@ -30,4 +35,6 @@
     elif (counter % 10 == 0 and counter > 0):
         time.sleep(2)
         print(f'\nResting for 2 seconds before we resume again \n')
+print(f'Old account subscribed to {len(subreddits_old_account)} subreddits.\n\n')
```

Comment:
I think this line is redundant since we aren't doing anything with the old subreddit account